### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by mikolajsobolewski**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+SHELL=/bin/bash
+
+UID := $(shell id -u)
+CURRENT_DIR := $(shell pwd)
+
+.PHONY: lint
+
+lint:
+	@docker run --rm --workdir /charts --volume "$(CURRENT_DIR):/charts" -u $(UID) -e "HOME=/tmp" quay.io/helmpack/chart-testing:v3.11.0 sh -ac 'chown ${UID} .; exec ct lint'
```
[the remark this answers is not in the document]
Comment (mikolajsobolewski) [reply Makefile:9]: it not needs, log output the same.

File: .github/workflows/test-charts.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          --config ct.yaml
+          --target-branch ${{ github.event.repository.default_branch }}
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.9.0
+        if: steps.list-changed.outputs.changed == 'true'
+
+      - name: Run chart-testing (install)
+        run: >
+          ct install
```
[the remark this answers is not in the document]
Comment (mikolajsobolewski) [reply .github/workflows/test-charts.yaml:55]: yes so we need correct default values

[the remark this answers is not in the document]
Comment (jlehtimaki) [reply .github/workflows/test-charts.yaml:55]: What about secrets on some deployment scenarios where those are needed?
Or does is successful when Kubernetes server accepts the manifest?

[the remark this answers is not in the document]
Comment (mikolajsobolewski) [reply .github/workflows/test-charts.yaml]: You're right we will have lots problems with secrets. May be return to this in future.

**Pull request merged**